### DISCUSSION TITLE
После обновления на 108 ошибки при сохранения поста

### DISCUSSION THREAD

**Discussion by erak31** [maxsite/cms#454] [General]
Доброго времени суток.
При сохранении поста (Опубликовать) вылазит куча нотисов с содержимым:

> A PHP Error was encountered
Severity: 8192

Message: The each() function is deprecated. This message will be suppressed on further calls

Filename: libraries/Xmlrpc.php

Line Number: 1385

A PHP Error was encountered
Severity: Notice

Message: Resource ID#150 used as offset, casting to integer (150)

Filename: libraries/Xmlrpc.php

Line Number: 706

A PHP Error was encountered
Severity: Notice

Message: Resource ID#150 used as offset, casting to integer (150)

Filename: libraries/Xmlrpc.php

Line Number: 707

Версия PHP 7.4

**Comment by maxsite** [comment]
Какой-то старый плагин. Возможно пинги. Отключите его.

**Reply by erak31** [reply]
Боже... вот я лох, ну конечно же он. всякие отключал, а его пропустил. Спасибо!!!
